pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>1.5.5.RELEASE</version>
+        <version>2.0.1.RELEASE</version>
```

Comment:
Have we discussed and decided on lifting all Eiffel components to the new Spring 2.0 version?